transactions/bitcoin/index.ts

```diff
@@ -32,22 +33,55 @@ export type {
   TransactionFeeOutput,
   TransactionOutput,
   TransactionScriptOutput,
+  TransactionSummary,
 };
 
 /**
  * send max bitcoin
  */
-export const sendMaxBtc = async (context: TransactionContext, toAddress: string, feeRate: number) => {
-  const paymentUtxos = await context.paymentAddress.getUtxos();
+export const sendMaxBtc = async (context: TransactionContext, toAddress: string, feeRate: number, skipDust = true) => {
```

Comment:
could we get unit tests on this? or do they exist already?

Reply:
I actually need to add tests for this entire file 👍 Thanks for catching